website/docs/getting-started/for-engineers.md

```diff
@@ -70,37 +70,157 @@ app.import('node_modules/@hashicorp/design-system-components/dist/styles/@hashic
 
 !!! Warning
 
-**Ensure a box-sizing reset is present**
+##### Ensure a box-sizing reset is present
```

Comment:
Should this semantically be a headline? There's also some styling applied to bold/strong text in these banners to match the color.

Reply:
I'd +1 this. It looks like it should be a heading here?

Reply:
No, this is the syntax used to add "titles" to the "banner" elements using markdown syntax.

Reply:
What I was actually trying to say is this _shouldn't_ be a headline within the banner.